web/styles/left_sidebar.css

```diff
@@ -1465,13 +1465,13 @@ li.top_left_scheduled_messages {
     }
 }
 
-#streams_list ul.topic-list {
+ul.topic-list {
```

Comment:
@amanagr FYI; this effectively reverts one of your prep commits, so you'll likely need to adjust something in the new PR.